### PR TITLE
Added graph coloring algorithm inside Greedy 

### PR DESCRIPTION
**Pull request by ritikverma2000**
**Graph coloring problem is to assign colors to certain elements of a graph subject to certain constraints. Vertex coloring is the most common graph coloring problem. The problem is, given m colors, find a way of coloring the vertices of a graph such that no two adjacent vertices are colored using the same color.
Chromatic Number: The smallest number of colors needed to color a graph G is called its chromatic number.
So basically my code finds the chromatic number for the given graph.**

### CODE REVIEW

**Comment by ritikverma2000** [comment]
@ericklarac if you could pls check my code and merge it in your repo it would be great as it an urgent requirement for me.

**Comment by keon** [comment]
Closing as stale — the repo was completely restructured in v1.0.0. Thank you for the contribution!
